[BUG] OnboardingFragment init fails on release builds

In issue #845 @msfjarvis adviced me to use the snapshot build from README.
I used the free one and received the following stacktrace when opening the app:

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{dev.msfjarvis.aps/com.zeapo.pwdstore.ui.onboarding.activity.OnboardingActivity}: android.view.InflateException: Binary XML file line #7 in dev.msfjarvis.aps:layout/activity_onboarding: Binary XML file line #7 in dev.msfjarvis.aps:layout/activity_onboarding: Error inflating class androidx.fragment.app.FragmentContainerView
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
Caused by: android.view.InflateException: Binary XML file line #7 in dev.msfjarvis.aps:layout/activity_onboarding: Binary XML file line #7 in dev.msfjarvis.aps:layout/activity_onboarding: Error inflating class androidx.fragment.app.FragmentContainerView
Caused by: android.view.InflateException: Binary XML file line #7 in dev.msfjarvis.aps:layout/activity_onboarding: Error inflating class androidx.fragment.app.FragmentContainerView
Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.zeapo.pwdstore.ui.onboarding.fragments.WelcomeFragment: make sure class name exists
	at androidx.fragment.app.FragmentManager$3.loadFragmentClass(FragmentManager.java:3)
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:2)
	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:4)
	at androidx.fragment.app.FragmentContainerView.<init>(FragmentContainerView.java:16)
	at androidx.fragment.app.FragmentLayoutInflaterFactory.onCreateView(FragmentLayoutInflaterFactory.java:3)
	at androidx.fragment.app.FragmentActivity.onCreateView(FragmentActivity.java:4)
	at android.view.LayoutInflater.tryCreateView(LayoutInflater.java:1069)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:997)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:961)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1123)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1084)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:682)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:534)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:481)
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:11)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:2)
	at androidx.activity.ComponentActivity.onCreate(ComponentActivity.java:19)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:1)
	at com.zeapo.pwdstore.ui.onboarding.activity.OnboardingActivity.onCreate(OnboardingActivity.kt:1)
	at android.app.Activity.performCreate(Activity.java:7824)
	at android.app.Activity.performCreate(Activity.java:7813)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
Caused by: java.lang.ClassNotFoundException: com.zeapo.pwdstore.ui.onboarding.fragments.WelcomeFragment
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:454)
	at androidx.fragment.app.FragmentManager$3.loadClass(FragmentManager.java:4)
	at androidx.fragment.app.FragmentManager$3.loadFragmentClass(FragmentManager.java:1)
	... 33 more
Caused by: java.lang.ClassNotFoundException: com.zeapo.pwdstore.ui.onboarding.fragments.WelcomeFragment
	... 37 more
```

Using the nonfree one resulted in the same traceback

--- a/app/src/main/java/com/zeapo/pwdstore/ui/onboarding/fragments/WelcomeFragment.kt
+++ b/app/src/main/java/com/zeapo/pwdstore/ui/onboarding/fragments/WelcomeFragment.kt
@@ -7,12 +7,14 @@ package com.zeapo.pwdstore.ui.onboarding.fragments
 
 import android.os.Bundle
 import android.view.View
+import androidx.annotation.Keep
 import androidx.fragment.app.Fragment
 import com.zeapo.pwdstore.R
 import com.zeapo.pwdstore.databinding.FragmentWelcomeBinding
 import com.zeapo.pwdstore.utils.performTransactionWithBackStack
 import com.zeapo.pwdstore.utils.viewBinding
 
+@Keep
 @Suppress("unused")
 class WelcomeFragment : Fragment(R.layout.fragment_welcome) {
 

